app/views/shared/dialogs/_dialog_field_drop_down_list.html.haml

```diff
@@ -19,7 +24,12 @@
   - if field.auto_refresh
     :javascript
       dialogFieldRefresh.listenForAutoRefreshMessages("#{field.id}", function() {
-        var selectedValue = $('select[name="#{field.name}"]').val();
+        var field = $('select[name="#{field.name}"]').length;
+        if (field) {
+          var selectedValue = $('select[name="#{field.name}"]').val();
+        } else {
+          var selectedValue = $('select[name="#{field.name}[]"]').val();
+        }
```

Comment:
In that case, we could probably use `$('select.dynamic-drop-down-#{field.id}').val();` to eliminate the conditional. I'm just thinking that there has to be something in common with both of these select boxes that makes it so we don't need to do this logic.